cli/storage_profile.c

```diff
@@ -221,19 +221,18 @@ perform_profile_run(
 {
     struct storage_profile_work *work_specs;
     struct storage_prof_stat stats;
-    struct itimerval timer = {0};
     struct statfs sbuf;
-    const uint32_t samples_per_thread = (file_sz / block_sz);
-    int      i, j, rc = 0;
-    bool     tmpfs;
+    double tmp, var_sum, mean, sigma;
     uint64_t sum, tot_samples;
-    double   tmp, var_sum;
-    double   mean;
-    double   sigma;
+    const uint32_t samples_per_thread = (file_sz / block_sz);
+    int i, j, rc = 0;
+    bool tmpfs;
     sigset_t blockset, origset;
 
     sigemptyset(&blockset);
     sigaddset(&blockset, SIGALRM);
+    sigaddset(&blockset, SIGINT);
+    sigaddset(&blockset, SIGTERM);
```

Comment:
So now I cannot interrupt the program with CTRL-C?  Why?

Reply:
We still can.. This is to just block these signals in the spawned threads. We restore the mask in the main thread immediately after the threads are created.